draft-tschofenig-rats-psa-token.md

```diff
@@ -531,7 +536,22 @@ with previous versions of the PSA attestation token format.
 {::include cddl/psa-profile.cddl}
 ~~~
 
-# Backwards Compatibility Considerations
+#### URI Structure for the Profile Identifier
+{: #sec-profile-uri-structure}
+
+A new profile is associated with a unique string.
+
+The string MUST use the URI fragment syntax defined in {{Section 3.5 of !RFC3986}}.
+
+The string SHOULD be short to avoid unnecessary overhead.
+
+To avoid collisions, profile authors SHOULD communicate upfront their intent to use a certain string using the enquiry form on the {{PSACertified}} website.
+
+To derive the value to be used for the `eat_profile` claim, the string is added as a fragment to the `tag:psacertified.org,2023:psa` tag URI {{!RFC4151}}.
+
+For example, an hypothetical profile using a COSE_Mac0 with AES Message Authenticationm Code (AES-MAC) may decide to use the string "s+aes-mac".  The `eat_profile` value would than be: `tag:psacertified.org,2023:psa#s+aes-mac`.
+
+## Backwards Compatibility Considerations
```

Comment:
Just checking: switching from `#` to `##` was intentional?

In this section there is a sentence `The new profile introduces three further changes:`
Should we say `The new profiles introduce three further changes:`?

Reply:
> Just checking: switching from `#` to `##` was intentional?

yes, I've moved this under §4 (PSA Claims).  However, I'm not sure anymore whether it's worth keeping all this legacy, which keeps piling up.  We might as well decide to start from scratch with this and point to previous versions of the draft for history.

> 
> In this section there is a sentence `The new profile introduces three further changes:` Should we say `The new profiles introduce three further changes:`?